doc/api/http.md

```diff
@@ -2059,6 +2062,9 @@ changes:
   * `ServerResponse` {http.ServerResponse} Specifies the `ServerResponse` class
     to be used. Useful for extending the original `ServerResponse`. **Default:**
     `ServerResponse`.
+  * `maxHeaderSize` {number} Optionally overrides the value of
+    [`--max-http-header-size][] for requests received by this server.
+    **Default:** 8KB.
```

Comment:
Doc the units: bytes. Also, describe default as `8192 (8KB)`, so that people don't set the value to `10`.